data_steward/analytics/cdr_ops/controlled_tier_qc/code/controlled_tier_qc.py

```diff
@@ -88,20 +88,26 @@ def display_check_summary_by_rule(checks_df, to_include):
         nothing_to_report = set(to_include) - set(by_rule['rule'])
 
         by_rule = by_rule.merge(check_description, how='outer', on='rule')
+
+        by_rule['n_row_violation'] = by_rule['n_row_violation'].fillna(
+            0).astype(int)
+
```

Comment:
Moved from below so changes to 'note' would be together. I did the same in the else statement.